rb/lib/selenium/webdriver/edge/legacy_support.rb

```diff
@@ -66,17 +64,18 @@ def mouseUp
         end
 
         def mouseMoveTo(element, x = nil, y = nil)
-          params = { :element => element }
+          params = { element: element }
 
           if x && y
-            params.merge! :xoffset => x, :yoffset => y
+            params[:xoffset] = x
+            params[:yoffset] = y
           end
 
           execute :mouseMoveTo, {}, params
         end
 
         def sendKeysToActiveElement(key)
-          execute :sendKeysToActiveElement, {}, :value => key
+          execute :sendKeysToActiveElement, {}, {value: key}
```

Comment:
I'm fine with anyone, up to you.
On Пн, 7 марта 2016 г. at 2:22, Titus <notifications@github.com> wrote:

> In rb/lib/selenium/webdriver/edge/legacy_support.rb
> <https://github.com/titusfortner/selenium/pull/1#discussion_r55147438>:
>
> >            end
> >
> >            execute :mouseMoveTo, {}, params
> >          end
> >
> >          def sendKeysToActiveElement(key)
> > -          execute :sendKeysToActiveElement, {}, :value => key
> > +          execute :sendKeysToActiveElement, {}, {value: key}
>
> @p0deje <https://github.com/p0deje> - Style/BracesAroundHashParameters -
> we can either do no_braces which removes these from all at the end, or
> context_dependent where it will put braces around the last parameters if
> a previous parameter was a hash. Current code is a little inconsistent, but
> tended toward the former. I think I prefer the latter. Which would you
> prefer?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/titusfortner/selenium/pull/1/files#r55147438>.
>
-- 
Alex